### DISCUSSION TITLE
Exchange Integration

### DISCUSSION THREAD

**Discussion by Wavesolid** [Aguafrommars/TheIdServer#1210] [Q&A]
I have a question regarding implementation WsFederation using this library, but why whenever the IDP redirect back to `/owa` the Exchange showed up an error 440. I tried the same with ADFS, have no issue with error code and performed succesfully login. Do you have any idea regarding this issue?

**Comment by aguacongas** [comment]
Why `/owa` ? The Ws-Federation endpoint is listening at `/wsfederation` and the metada document can be retrieved at `/wsfederation/metadata`. Read https://github.com/Aguafrommars/TheIdServer/tree/master/src/IdentityServer/Duende/Aguacongas.IdentityServer.WsFederation.Duende for more information.
